packages/firestore/test/unit/local/remote_document_cache.test.ts

```diff
@@ -76,11 +83,31 @@ function genericRemoteDocumentCacheTests(): void {
       });
   }
 
+  function assertMatches<V>(
```

Comment:
What is V?

Reply:
The value... which is now unused. This used to take a Map<DocumentKey, V> type.